### DISCUSSION TITLE
Sort Functionality

### DISCUSSION THREAD

**Discussion by MiteshreeBarolia** [microsoft-search/pnp-modern-search#3416] [General]
We want to implement sort functionality in pnp search webparts. We want to sort the result data on basis of Title. We tried with the default Title column, by managed property by mapping crawled property of title. But the resulted data did not get sorted in any order. We tried by passing different query to check that is it not working with any of our list/library but came out to be its not even working for the deafult query {search Term}.

**Comment by kasperbolarsen** [comment]
It most certainly is possible. On each tenant I add this mapping:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/d2a3fc19-44f2-495b-86bc-fa6e19dd9add)

Once the mapping has been completed and values starts showing up in the RefinableString, I add it to the "Selected Properties" in the Result web part, configure the sort in "Edit Sort Settings", and adds the RefinableString as the sort option in "Manage Columns" (select sort field).

If it just will not work, I can only recommend signing up for a 15-30 minute session in PnP Office Hours, as per https://github.com/microsoft-search/pnp-modern-search/ ( scroll down)

**Comment by MiteshreeBarolia** [comment]
We have followed the above steps and tried the sorting, but the data is not getting sorted.
